backend/ml/mehestan/parameters.py

```diff
@@ -0,0 +1,8 @@
+from solidago.pipeline import PipelineParameters
+
+from tournesol.utils.constants import COMPARISON_MAX, MEHESTAN_MAX_SCALED_SCORE
+
+
+class MehestanParameters(PipelineParameters):
+    r_max = COMPARISON_MAX
+    max_squashed_score = MEHESTAN_MAX_SCALED_SCORE
```

Comment:
(1) As this is extending the dataclass PipelineParameters, I think it should include the type annotation.

```class MehestanParameters(PipelineParameters):
    r_max: float = COMPARISON_MAX
    max_squashed_score: float = MEHESTAN_MAX_SCALED_SCORE
```

(2) Shouldn't we define this type in Solidago and only use it in Tournesol backend